com.unity.cinemachine/Runtime/Timeline/CinemachinePlayableMixer.cs

```diff
@@ -8,8 +8,10 @@ namespace Unity.Cinemachine
 {
     internal sealed class CinemachinePlayableMixer : PlayableBehaviour
     {
-        public delegate PlayableDirector MasterDirectorDelegate();
+        /// <summary>The priority of the track.  Higher numbers override lower ones.</summary>
```

Comment:
Extra space

Reply:
where?  I don't  understand.

Reply:
Antoine means the double space at the end of sentences in the summary tag.

We do that for tooltips usually, because it looks better.

Reply:
I always always always do that.  Nothing will make me stop.